.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/Shlomi.Benita/.pyenv/shims/python"
```

Comment:
This points to a local env setting, can this be removed?

Reply:
Removed. sorry.